tests/lora/test_lora_layers_flux.py

```diff
@@ -162,6 +163,56 @@ def test_with_alpha_in_state_dict(self):
         )
         self.assertFalse(np.allclose(images_lora_with_alpha, images_lora, atol=1e-3, rtol=1e-3))
 
+    def test_lora_expansion_works_for_absent_keys(self):
```

Comment:
`test_lora_expansion_works_for_extra_keys`? we could add the original case `test_lora_expansion_works_for_absent_keys`

Reply:
That works for me.